### PR TITLE
update version to 0.2.0

### PR DESCRIPTION
**Pull request by LakshK98**
## Description

- Updated version to 0.2.0

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
   <PropertyGroup>
```
Comment (matthewige) [inline Directory.Build.props:12]: This looks like the previous PR that was made to update the version - https://github.com/microsoft/ntosebpfext/pull/111/files

Do we need to update this?

**Review by matthewige — APPROVED** [review]

File: Directory.Build.props (hunk 1 above)
Comment (LakshK98) [reply Directory.Build.props:12]: added ebpf_ext_version.h update to the script

**Review by matthewige — APPROVED** [review]

File: Directory.Build.props [hunk 2]
```diff
@@ -11,7 +11,7 @@
...
     <eBPFExtensionsVersionMajor>0</eBPFExtensionsVersionMajor>
-    <eBPFExtensionsVersionMinor>1</eBPFExtensionsVersionMinor>
```
Comment (shankarseal) [inline Directory.Build.props:14]: why not only have the script in this PR; and follow up with a second PR generated by the script to bump up the version?

**Review by matthewige — APPROVED** [review]

File: Directory.Build.props (hunk 2 above)
Comment (LakshK98) [reply Directory.Build.props:14]: moved update script to another pr

**Pull request merged**